src/site/content/en/blog/bbc-hsts/index.md

```diff
@@ -0,0 +1,74 @@
+
+HSTS is dictated by the `Strict-Transport-Security` HTTP response header for HTTPS requests. When set, return visits to a website will trigger a special redirect known as a "307 Internal Redirect", which is when the browser handles the redirect logic. These types of redirect are extremely fast, and add no noticeable latency during an HTTP-to-HTTPS hop.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/J9vQLJCLmnHQPb6spsRe.png", alt="A 307 internal redirect from HTTP to HTTPS, triggered by an HSTS header. The 307 redirect only takes 2 milliseconds.", width="800", height="97" %}
+</figure>
+
+Similar in syntax to `Cache-Control`'s `max-age` directive, an HSTS header specifies a `max-age` directive. This directive takes a value in seconds that specifies how long the policy should take effect:
+
+```http
+Strict-Transport-Security: max-age=3600
+```
+
+In the above example, the policy should only take effect for an hour.
+
+## Deploying HSTS
+
+The main drawback of deploying HSTS is if you're not ready to treat your origin as strictly secure. Let's say you have a number of subdomains you're serving resources from, but maybe not all of them are secure. In this scenario, an HSTS header could break your website.
+
+The BBC took the right approach to deploying HSTS. [As Neil Craig said in his tweet](https://twitter.com/tdp_org/status/1549022691772059649), the initial value that was set for bbc.com was `max-age=10`.
+
+This approach means that the policy was only initially effective for ten seconds. This doesn't provide much of a benefit, but the idea is to feel out whether there may be issues with applying HSTS at all. As time goes on, you can increase the policy incrementally and see if issues occur. At present, bbc.com is specifying an HSTS policy of `max-age=3600`, and that will almost certainly increase.
+
+You certainly don't want to come out of the gate with a long `max-age` value when deploying HSTS. You could find yourself suddenly scrambling to fix issues while users experience issues. Start small, and increment over time! When you're confident, you can set your `max-age` directive to a much longer period of time.
+
+## Get faster initial navigations with the HSTS preload list
+
+An HSTS policy only takes effect after the first visit to a website, so the benefit isn't noticeable until a user comes back. However, you can _preload_ your HSTS policy by submitting your website to the [HSTS preload list](https://hstspreload.org/), which is a hardcoded list of websites that the browser knows are strictly HTTPS. When your site is on the preload list, HTTP-to-HTTPS redirect latency via HSTS will be instantaneous&mdash;**even on the first visit!**
+
+{% Aside %}
+If you have a domain with a .dev TLD, your website is required to have a valid SSL certificate to function at all. This means you don't need to add your .dev website to the HSTS preload list&mdash;it will already be assumed as strictly secure by the browser!
```

Comment:
Don't like the implication that SSL implies HSTS (it's the other way around).

```suggestion
If you have a domain with a `.dev` top level domain (TLD), your website is already protected by HSTS as the whole `.dev` TLD has been included in preload list. It will already be assumed as strictly secure by the browser and is required to have a valid SSL certificate to function at all.
```